src/ngraph/builder/quantization.cpp

```diff
@@ -105,116 +105,114 @@ namespace ngraph
                                            const Strides& data_dilation_strides,
-                                           const std::shared_ptr<Node> min_input,
-                                           const std::shared_ptr<Node> max_input,
-                                           const std::shared_ptr<Node> min_filter,
-                                           const std::shared_ptr<Node> max_filter,
-                                           const std::shared_ptr<Node> min_freezed_output,
-                                           const std::shared_ptr<Node> max_freezed_output,
+                                           std::shared_ptr<Node> min_input,
+                                           std::shared_ptr<Node> max_input,
+                                           std::shared_ptr<Node> min_filter,
+                                           std::shared_ptr<Node> max_filter,
+                                           std::shared_ptr<Node> min_freezed_output,
+                                           std::shared_ptr<Node> max_freezed_output,
                                            const bool with_relu)
         {
-            float scale = builder::quantization_util::get_scale(min_input,
-                                                                max_input,
-                                                                min_filter,
-                                                                max_filter,
-                                                                min_freezed_output,
-                                                                max_freezed_output);
-            auto requantization_scale = op::Constant::create(element::f32, Shape{1}, {scale});
-
-            return make_shared<op::QuantizedConvolutionBias>(data_batch,
+            auto scale = quantization_util::get_scale(min_input,
+                                                      max_input,
+                                                      min_filter,
+                                                      max_filter,
+                                                      min_freezed_output,
+                                                      max_freezed_output);
+
+            return make_shared<op::QuantizedConvolutionBias>(input,
                                                              filters,
                                                              bias,
                                                              window_movement_strides,
                                                              window_dilation_strides,
                                                              padding_below,
                                                              padding_above,
                                                              data_dilation_strides,
-                                                             requantization_scale,
+                                                             scale,
```

Comment:
I think requantization_scale is a more appropriate name because for QC's this scale is used for requantizing from i32 to {u}i8